Jenkinsfile

```diff
@@ -25,7 +25,7 @@ pipeline {
         anyOf { buildingTag() }
```

Comment:
should we rename this stage `Build & Test images`?

Reply:
I was leaving the test phase out of CI on purpose (as per https://github.com/netlify/pod-workflow/issues/137#issuecomment-917057034 as well as https://github.com/netlify/pod-workflow/issues/251) , so this stage is still only building the actual `Build Image`. I was hoping we could focus on bringing these tests to the CI env on a separate PR?